OpenRA.Mods.Common/Traits/Player/ClassicParallelProductionQueue.cs

```diff
@@ -0,0 +1,223 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Attach this to the player actor (not a building!) to define a new shared build queue.",
+		"Will only work together with the Production: trait on the actor that actually does the production.",
+		"You will also want to add PrimaryBuildings: to let the user choose where new units should exit.")]
```

Comment:
There should be a description of how the build time is affected by number of production facilities and number of parallel production items since it is not a straight forward product of the two multipliers.